codecarbon/core/cpu.py

```diff
@@ -209,3 +211,36 @@ def get_cpu_details(self) -> Dict:
                 exc_info=True,
             )
         return cpu_details
+
+
+class TDP:
+    def __init__(self):
+        self.tdp = self._get_power_from_constant()
+
+    def _parse_cpu_model(self, raw_name) -> str:
+        """
+        Parse the model name from the raw name extracted from cpuinfo library
+        :return: parsed CPU name
+        """
+        return (
+            raw_name.split(" @")[0]
+            .replace("(R)", "")
+            .replace("(TM)", "")
+            .replace(" CPU", "")
+        )
```

Comment:
I don't think this function needs to be a method, it's not using `self`, let's extract it and add at least one test to check that's it parsing correctly the CPU model.